KsApi/models/ErrorEnvelope.swift

```diff
@@ -134,6 +135,22 @@ public struct ErrorEnvelope {
       exception: nil
     )
   }
+
+  /**
+   An error that that occurs when the current checkout is invalid.
+
+   - parameter message: The error message returned from the server.
+
+   - returns: An error envelope that describes why the validation failed.
+   */
+  internal static func validateCheckoutError(_ message: String) -> ErrorEnvelope {
+    return ErrorEnvelope(
+      errorMessages: [message],
+      ksrCode: .ValidateCheckoutError,
+      httpCode: 200,
```

Comment:
nit: 200 seems like a weird choice for the http code for an error

Reply:
Well, in this case, technically, the request was sent successfully even though there was an error on the backend. A 400 (Bad Request) didn't make much sense to me either.